### PR TITLE
Fetch Data from Upstream

### PR DESCRIPTION
**Pull request by maennchen**

### CODE REVIEW

File: lib/data/countries.yaml [hunk 1]
```diff
@@ -166,7 +165,7 @@
 - NG
 - NI
 - NL
-- 'NO'
+- NO
```
Comment (SebastianSzturo) [inline lib/data/countries.yaml]: I think this will result as “false” in yaml

Comment (maennchen) [reply lib/data/countries.yaml]: @SebastianSzturo According to the test that I just committed, `NO` works fine.

Comment (SebastianSzturo) [reply lib/data/countries.yaml]: Oh perfect, thanks for checking! I'll give this another spin later today and release a new version. Thank you for the contribution 🙏

**Comment by dideler** [comment]
Hey all, I'm wondering how a user of this library would go about updating the country data? Had a quick look and couldn't find a task to do so.

For this PR for example, did you copy the country files from the [countries gem v2.1.4](https://github.com/hexorx/countries/tree/v2.1.4)?

**Comment by maennchen** [comment]
@dideler Yes, I copied over the new data from the gem.

**Comment by maennchen** [comment]
@SebastianSzturo Ping

**Comment by maennchen** [comment]
Ping

**Comment by maennchen** [comment]
Closed because of inactivity

**Pull request closed without merging**